tests/test_client/abstract_client.rs

```diff
@@ -0,0 +1,329 @@
+// Copyright 2019 Contributors to the Parsec project.
```

Comment:
2020 I think 😁

Reply:
Ok, I wasn't sure because I thought I saw you changing some to `2019`

Reply:
I should have changed all ranges `2019 - 2020` to `2019` but kept `2020` to `2020`. If not I made a mistake!